lib/chkname.h

```diff
@@ -19,8 +20,14 @@
  *   false - bad name
  */
 
-#include "defines.h"
 
+#include <config.h>
+
+#include <stdbool.h>
+#include <stddef.h>
+
+
+extern size_t login_name_max_size(void);
```

Comment:
For 989 PR we would require similar function for max group size.
The issue with coverity report error is to point out in the code that any group name passed do SYSLOG will be explicitly limited to some length.